.github/workflows/build-release.yml

```diff
@@ -13,16 +13,19 @@ env:
 jobs:
   build-release:
     name: Build and push tag images
-    strategy: 
+    strategy:
       matrix:
         architecture: [amd64,ppc64le,arm64,s390x]
         platform: [linux]
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v4
-    - name: Set Env Tags
+    - name: Set Release Tag
```

Comment:
Why is this changing?

Reply:
This is changing for clarity, since we are setting multiple ENVs, if the step for the commit goes away, this can be reverted.

Reply:
Sorry, I misread which was the incoming change and which was outgoing. This looks fine to me.